search.py

```diff
@@ -61,10 +61,10 @@ def reindex_app(app):
     machine_class, tasks = SEARCHABLE_APPS[app]
     util.use_random_host('class-%s' % machine_class)
 
-    for task in tasks:
+    for t in tasks:
```

Comment:
@MatMoore I'd usually avoid one-character variable names, however given the limited scope and close locality I'm not sure it's an issue? I'm hard-pressed to come up with a better name but open to suggestions.